.ci/spellcheck/.pyspelling.wordlist.txt

```diff
@@ -515,3 +515,9 @@ YOLO
 YOLOv
 yolov
 Zongyuan
+OneFormer
```

Comment:
generally we prefer to add words preserving alphabetical order (to easily track simmilar words and reduce number of conflicts if several PRs update spellcheking vocab in parallel)